### PR TITLE
replay(feat): Make the url walker hovercards consistent

### PR DESCRIPTION
**Pull request by ryan953**
Make the hover state for the url walker consistent, whether there's one url in the segment, or many.

| | Replay Index (no links) | Replay Details (timestamp link) |
| --- | --- | --- |
| Before | before - table | before - links |
| After | after - table | after - links |

### CODE REVIEW

File: static/app/components/replays/walker/splitCrumbs.tsx [hunk 1]
```diff
@@ -134,10 +106,14 @@ function SummarySegment({
     </ScrollingList>
   );
 
+  const label =
+    crumbs.length === 1
+      ? getUrl(first(crumbs))
```
Comment (scttcper) [inline static/app/components/replays/walker/splitCrumbs.tsx:111]: you checked the length
```suggestion
      ? getUrl(crumbs)
```

File: static/app/components/replays/walker/urlWalker.tsx [hunk 2]
```diff
@@ -62,5 +57,5 @@ function urlToCrumb(url: string) {
     color: 'green300',
     timestamp: undefined,
     data: {to: url},
-  } as BreadcrumbTypeNavigation;
+  } as Crumb;
```
Comment (scttcper) [inline static/app/components/replays/walker/urlWalker.tsx:60]: just as another nit pick

prefer return types to casting if possible

Comment (ryan953) [reply static/app/components/replays/walker/urlWalker.tsx:60]: NP

File: static/app/components/replays/walker/splitCrumbs.tsx (hunk 1 above)
Comment (billyvg) [reply static/app/components/replays/walker/splitCrumbs.tsx:111]: You can index access though and drop the lodash dep

**Review by billyvg — APPROVED** [review]

**Review by eliashussary — APPROVED** [review]

**Pull request merged**